db/seeds.rb

```diff
@@ -1,7 +1,6 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the rake db:seed (or created alongside the db with db:setup).
-#
-# Examples:
-#
-#   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
-#   Mayor.create(name: 'Emanuel', city: cities.first)
+# Создадим записи в базе данных для проверки, восклицательный знак после create стоит для вызова исключения в случае ошибки создания записи.
+Card.create!([
+  {original_text:"Привет. Как дела?", translated_text:"Hello. How are you?", review_date: Date.current},
+  {original_text:"Откуда ты?", translated_text:"where are you from?", review_date: Date.yesterday},
+  {original_text:"Привет. Как дела?", translated_text:"Hello. How are you?", review_date: Date.new(2015, 02, 12)}
+  ])
```

Comment:
Indent the right bracket the same as the start of the line where the left bracket is.